src/Home/index.tsx

```diff
@@ -69,7 +73,10 @@ const Home: FunctionComponent = () => {
         ) : (
           <EscrowVerificationFlowButton />
         )}
-        <ShareLink />
+        {healthAuthorityHealthCheckUrl && (
+          <HealthCheckLink healthCheckUrl={healthAuthorityHealthCheckUrl} />
+        )}
+        {appDownloadUrl && <ShareLink appDownloadUrl={appDownloadUrl} />}
```

Comment:
pulled this up to keep a consistent layer of abstraction for the view logic on the dashboard.